lib/ex_geo/downloader.ex

```diff
@@ -8,11 +8,28 @@ defmodule ExGeo.Downloader do
   """
   @spec download!(binary) :: binary
   def download!(url, retry \\ 1)
-  def download!(_, retry) when retry >= 3, do: raise DownloadError, message: "retries exceeded"
+  def download!(_, retry) when retry >= 3, do: raise(DownloadError, message: "retries exceeded")
+
   def download!(url, retry) do
     case HTTPoison.get(url) do
       {:ok, %HTTPoison.Response{status_code: 200, body: body}} -> body
       _ -> download!(url, retry + 1)
     end
   end
-end
\ No newline at end of file
+end
+
+defmodule ExGeo.Downloader.MaxmindHelper do
```

Comment:
Maybe make this a serializer behavior, and have a default one configure that resolves to an identity function.

Reply:
How do you mean serializer behaviour?

Reply:
Oh basically what you made but as a behaviour, I just thought that would be a good name because I couldn't think of anything